### ISSUE TITLE
Undo new snow texture

### ISSUE THREAD

**Issue by mwrsa** [minetest/minetest_game#1289]
While I like the gleam in the new snow texture, the uniform waves (very realistic at that) make my eyes hurt. Consider changing it back please. IDK if others will join in here, but on the server people share this opinion. I thought I'd put a voice to it.

**Comment by RHRhino** [comment]
@mwrsa If you don't like the new snow texture, then use a texture pack with the old texture. The new snow texture is much better than the old one and should be kept as default.

**Comment by KatiHlov** [comment]
I like the new texture as well, but, it only looks good on small areas like one or two nodes, when you are near a huge amount of snow/snowblocks it looks like a kind of white sand or even zebra print, and let's don't talk about the sides of snowblocks.

I personally don't mind using a texture pack, that's why they were made, but I wouldn't use it for any building proposes and I guess some people may do the same just to avoid confusions.

Maybe just a small retouch making it less noticeable would help

**Comment by everamzah** [comment]
I think reverting is too much, but enhancing the pattern and finessing the sides is probably wise.

>Maybe just a small retouch making it less noticeable would help

Is this something you can take a stab at?  This issue would make a fine place to discuss improving the current snow texture.

**Comment by KatiHlov** [comment]
The sides are visible even from the distance (I felt like a Lion trying to chase a lot of zebras), thing that it wouldn't happen on real snow, a piled snow would look kind of smooth. Making the dark lines would make it more versatile for projects that includes its.
![picsart2016-17-9--13-18-39](https://cloud.githubusercontent.com/assets/22256301/18610126/5d76bd94-7cd9-11e6-88fe-9e54b0472b02.png)
This is what I (me) think would be a good example for piled up snow (I don't own the image, it's from a Minecraft texture and I just used it as example)

**Comment by paramat** [comment]
We can possibly reduce the darkness of some of the 'wave lines', i'll give it a try.

**Comment by paramat** [comment]
Reducing the darkness of one of the wavy lines is not necessarily better, it creates more repetition because both lines are now the same darkness:

![screenshot_20160917_233954](https://cloud.githubusercontent.com/assets/3686677/18611419/4d5e0f5a-7d30-11e6-9653-701d91d4279c.png)

So i'm for now unsure about changing this classic Gambit texture.

**Comment by mwrsa** [comment]
The zebra comparison brings out the point I wanted to stress. Most screenshots also show altitude steps, but look from an angle where they are not visible or take a larger flat area in general, and your vision will blur too. It's like one of those 3D images where you have to cross your eyes to see them in 3D. Just that there is nothing to see in the snow here.
Also I apologize for just criticizing without offering a solution - that's what I'd rather like to do, but I have none.
To sum it up: Could you please link to the default texture somewhere? The wave isn't working for me. I already have a custom texture pack for such cases, like default gravel...
Thank you. 

As a totally not serious side note - ever considered adding yellow snow? xD

**Comment by paramat** [comment]
Gambit's original:
![default_snow_gambit](https://cloud.githubusercontent.com/assets/3686677/18620591/868756dc-7e0d-11e6-849b-0c93ca6c2034.png)

**Comment by Fixer-007** [comment]
I've played a lot with new texture, and I like it, it looks nice ingame.

**Comment by paramat** [comment]
Snce this request is to return to the previous texture i've added 'won't add'.
